Gemfile

```diff
@@ -40,6 +40,7 @@ gem 'graphql-pagination'
 # Payment processing
 gem 'gocardless_pro', '~> 2.34'
 gem 'stripe'
+gem 'adyen-ruby-api-library'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `adyen-ruby-api-library` should appear before `stripe`.